lib/include/openamp/virtio_rng.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef VIRTIO_RNG_H
+#define VIRTIO_RNG_H
+
+#include <openamp/virtqueue.h>
+#include <openamp/virtio.h>
+#include <metal/device.h>
+
+#define VQIN_SIZE 2
+
+
+/**
+ * @brief Initialize a VIRTIO entropy device.
+ *
+ * @param[in] vdev Pointer to virtio_device structure.
+ * @param[in] vqs Array of pointers to the virtqueues used by the device.
+ * @param[in] vq_names Array of pointers to the virtqueues names.
+ * @param[in] vq_count Number of virtqueues the device uses.
+ *
+ * @return int 0 for success.
+ */
+
+int virtio_mmio_rng_init(struct virtio_device *vdev, struct virtqueue **vqs,
```

Comment:
Could you supress mmio term?
At the end,  the virtio service should be  independent from the transport layer

Reply:
Done